.github/workflows/CI.yml

```diff
@@ -747,7 +747,8 @@ jobs:
       - name: run tests
         run: |
           export TEST_IMAGE=public.ecr.aws/${{ env.ECR_REPO }}:${{ needs.e2etest-preparation.outputs.version }}
-          cd test-framework/adot-testbed
+          cd testing-framework/adot-testbed
+          gradle wrapper
```

Comment:
why do you need this?

Reply:
the wrapper is already here: https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/adot-testbed/gradlew

So I'm not following why we need this step.

Reply:
In My local I am starting to get this

```
Error: Could not find or load main class org.gradle.wrapper.GradleWrapperMain
Caused by: java.lang.ClassNotFoundException: org.gradle.wrapper.GradleWrapperMain
```
So have added this command here as i was thinking it could be the same case here.